geps/gep-713.md

```diff
@@ -560,52 +578,567 @@ ties:
+An important note here is that a key piece of information for Policy Admins and
+Cluster Operators is “How many things does this Policy affect?”. In the parable,
+this would have enabled Charlie to know that deleting the Namespace Policy would
+affect many other people than just Ana.
+
+### Problems we need to solve
+
+Before we can get into solutions, we need to discuss the problems that solutions
+may need to solve, so that we have some criteria for evaluating those solutions.
+
+#### User discoverability
+
+Let's go through the various users of Gateway API and what they need to know about
+Policy Attachment.
+
+In all of these cases, we should aim to keep the troubleshooting distance low;
+that is, that there should be a minimum of hops required between objects from the
+one owned by the user to the one responsible for a setting.
+
+Another way to think of the troubleshooting distance in this context is "How many
+`kubectl` commands would the user need to do to understand that a Policy is relevant,
+which Policy is relevant, and what configuration the full set of Policy is setting?"
+
+##### Application Developer Discoverability
+
+How does Ana, or any Application Devloper who owns one or more Route objects know
+that their object is affected by Policy, which Policy is affecting it, and what
+the content of the Policy is?
+
+The best outcome is that Ana needs to look only at a specific route to know what
+Policy settings are being applied to that Route, and where they come from.
+However, some of the other problems below make it very difficult to achieve this.
+
+##### Policy Admin Discoverability
+
+How does the Policy Admin know what Policy is applied where, and what the content
+of that Policy is?
+How do they validate that Policy is being used in ways acceptable to their organization?
+For any given Policy object, how do they know how many places it's being used?
+
+##### Cluster Admin Discoverability
+
+The Cluster Admin has similar concerns to the Policy Admin, but with a focus on
+being able to determine what's relevant when something is broken.
+
+How does the Cluster Admin know what Policy is applied where, and what the content
+of that Policy is?
+
+For any given Policy object, how do they know how many places it's being used?
+
+#### Evaluating and Displaying Resultant Policy
+
+For any given Policy type, whether Direct Attached or Inherited, implementations
+will need to be able to _calculate_ the resultant set of Policy to be able to
+apply that Policy to the correct parts of their data plane configuration.
+However, _displaying_ that resultant set of Policy in a way that is straightforward
+for the various personas to consume is much harder.
+
+The easiest possible option for Application Developers would be for the
+implementation to make the full resultant set of Policy available in the status
+of objects that the Policy affects. However, this runs into a few problems:
+- The status needs to be namespaced by the implementation
```

Comment:
I wish our markdown parsing didn't require new lines here, but unfortunately it does and we constantly miss it because it's not intuitive. https://deploy-preview-2128--kubernetes-sigs-gateway-api.netlify.app/geps/gep-713/#evaluating-and-displaying-resultant-policy

```suggestion
of objects that the Policy affects. However, this runs into a few problems:

- The status needs to be namespaced by the implementation
```